src/ast/value.rs

```diff
@@ -45,6 +45,10 @@ pub enum Value {
     SingleQuotedByteStringLiteral(String),
     /// B"string value"
     DoubleQuotedByteStringLiteral(String),
+    /// R'string value' or r'string value'
+    SingleQuotedRawStringLiteral(String),
+    /// R"string value" or r"string value"
+    DoubleQuotedRawStringLiteral(String),
```

Comment:
@togami2864, this seems ok, but I want to raise a question.

Is the quote style of enormous importance here? Does it change the context to a level that the two variants will be handled individually? If `r'abc'` is different, semantically, from r"abc", that's fine. Otherwise, could only be held within a single variant, with an enum of quote styles? (It's a sincere question, if you don't think so, then we can keep it this way).

Reply:
The disadvantage is that the test becomes more complicated(due to canonical), but since there are no other disadvantages, I have merged the two values `RawStringLiteral`.